### PR TITLE
fix headers in L1Trigger/L1TMuonEndCap

### PR DESCRIPTION
**Pull request by mrodozov**
Add missing headers for passing check-headers scram check

### CODE REVIEW

**Comment by mrodozov** [comment]
please test

File: L1Trigger/L1TMuonEndCap/interface/bdt/LossFunctions.h [hunk 1]
```diff
@@ -139,8 +139,8 @@ class Huber : public LossFunction
         double target(Event* e) override
         {
         // The gradient of the loss function.
-
-            if (std::abs(e->trueValue - e->predictedValue) <= quantile)
+	  double multiplier = ((e->trueValue - e->predictedValue) > 0) ? 1.0 : -1.0;
+	  if ((e->trueValue - e->predictedValue)*multiplier <= quantile)
```
Comment (thomreis) [inline L1Trigger/L1TMuonEndCap/interface/bdt/LossFunctions.h]: What is the advantage over std::abs?

Comment (smuzaffar) [reply L1Trigger/L1TMuonEndCap/interface/bdt/LossFunctions.h]: @mrodozov , I do not understand the reason behind this change.

Comment (mrodozov) [reply L1Trigger/L1TMuonEndCap/interface/bdt/LossFunctions.h]: the compiler was complaining with an error not to use abs on a double. so I substituted the usage of abs with an multiplier

**Comment by mrodozov** [comment]
I'm closing this in favor of https://github.com/cms-sw/cmssw/pull/23188. The compiler complain was because of missing cmath header and not that there was anything wrong with abs(double).

**Pull request closed without merging**